Formula/rocksdb.rb

```diff
@@ -21,6 +21,7 @@ def install
     ENV["PORTABLE"] = "1"
     ENV["DEBUG_LEVEL"] = "0"
     ENV["USE_RTTI"] = "1"
+    ENV["ROCKSDB_DISABLE_ALIGNED_NEW"] = "1"
```

Comment:
If this only fails on Sierra, we probably want to write it as:

```ruby
ENV["ROCKSDB_DISABLE_ALIGNED_NEW"] = "1" if MacOS.version <= :sierra
```

Reply:
Excellent suggestion, thanks. Put it in and the formula built fine.